tests/api_fastapi/core_api/routes/public/test_task_instances.py

```diff
@@ -1027,20 +1028,26 @@ def test_should_respond_200_for_dag_id_filter(self, test_client, session):
         assert count == len(response.json()["task_instances"])
 
     @pytest.mark.parametrize(
-        "order_by_field", ["start_date", "logical_date", "data_interval_start", "data_interval_end"]
+        "order_by_field, date",
```

Comment:
```suggestion
        "order_by_field, base_date",
```
base_date might be better. `date` might shadow `from datetime import date`

Reply:
Implemented change